src/browser_action.js

```diff
@@ -498,6 +497,25 @@ browseraction.createEventDiv_ = function(event) {
   return eventDiv;
 };
 
+function getGCalUrl() {
+  return "https://calendar.google.com/calendar/";
+}
+
+function isGCalUrl(url) {
+  return url.indexOf(getGCalUrl()) == 0;
+}
+
+function goToCalendar(eventUrl) {
```

Comment:
Hmm, I think I prefer it this way. I often end up with multiple tabs I didn’t intend to open, which eventually annoys me because I start editing an event in one tab, and then switch away, and can’t find that tab any more after returning. More tabs also means Chrome requires more RAM.

To do the opposite, there’s already an easy way: simply open a new tab and start typing `calendar.google.com`, which is easy if that’s what you wanted.

So I’d prefer this change!